Pages/Shared/_Layout.cshtml

```diff
@@ -1,8 +1,7 @@
-﻿@{
+@{
     string title = "Motte Quality Vault";
     string pageTitle = ViewData["Title"] as string;
-    if (pageTitle != null && pageTitle != "")
-    {
+    if (pageTitle != null && pageTitle != "") {
```

Comment:
This all seems like it's unrelated; can this be undone for the sake of a minimal change?